setup.py

```diff
@@ -7,7 +7,7 @@
 _requirements = os.path.join(_base, 'requirements.txt')
 _requirements_test = os.path.join(_base, 'requirements-test.txt')
 
-version = "1.4.4"
+version = "1.4.5"
```

Comment:
v1.4.5 and v1.4.6 do already exist! Also I was wondering with all those changes if we should make it a minor version upgrade to 1.5.0. What do you think?